kaldb/src/test/java/com/slack/kaldb/server/KalDbIntegrationTest.java

```diff
@@ -0,0 +1,84 @@
+package com.slack.kaldb.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.slack.kaldb.config.KaldbConfig;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KalDbIntegrationTest {
+  private static final Logger LOG = LoggerFactory.getLogger(KalDbIntegrationTest.class);
+
+  private Kaldb kaldb;
+  private final ObjectMapper om = new ObjectMapper();
+
+  @Before
+  public void start() throws IOException {
+    KaldbConfig.reset();
```

Comment:
This is the same approach we use on `KaldbConfigTest.java`. Unfortunately we seem to have a significant amount of classes now that set state into `KaldbConfig.java` and don't cleanup on exit - including helper methods (ie, `KaldbConfigUtil.initEmptyIndexerConfig`). 

We may want to reconsider how we're making the config values available, as this will continue to be an issue.